package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "balena-cli",
-  "version": "10.17.5",
```

Comment:
I reckon we will need to publish this backport manually (npm & github releases), and versionbot will not automatically update the version for us.

Reply:
Moved this to its own 'v10.17.6' commit (alongside the changelog changes) to mimic what versionbot would do.